fgl.cabal

```diff
@@ -67,6 +68,7 @@ library {
     build-depends:    base < 5
                     , transformers
                     , array
+                    , vector
```

Comment:
Is there a good strategy to discover the bounds? I'm not honestly sure what they are.

Should I just search backwards through time in the vector library until the functions I use are no longer present?